relay_client/src/http.rs

```diff
@@ -280,7 +288,11 @@ impl Client {
         let status = result.status();
 
         if !status.is_success() {
-            return Err(HttpClientError::InvalidHttpCode(status).into());
+            let body = match result.text().await {
```

Comment:
I'd use `result.text().await.unwrap_or_else(|err| format!("... error calling result.text(): {err:?}"))`. Also the message is a bit strange. Maybe it should be something like 'error reading response body'?

Reply:
I don't know how I forgot about that function, fixed in [9729b6a](https://github.com/WalletConnect/WalletConnectRust/pull/41/commits/9729b6a209b304c75bf31cbc8ba71a5d51efcfba)

Three dots was suppose to clearly separate a body from a body error, but probably unnecessary.

Actually, I just pushed another commit that uses the Result value directly: [fec5ef6](https://github.com/WalletConnect/WalletConnectRust/pull/41/commits/fec5ef6dc67fa9fb37a3417d782839a0e13c14eb)